### ISSUE TITLE
[FR] Keyboard navigation in table grids

### ISSUE THREAD

**Issue by philiprenich** [AppFlowy-IO/AppFlowy#8309]
### Description

I would like to be able to tab or generally navigate data grids by keyboard. It is very slow to have to click with the mouse into a cell, then type, then use the mouse again, type, repeat...

If the rows are text only it seems to work, but not if you try to tab to a tag or date style cell.

Additionally, there is no way to exist a tag or date cell aside from `esc` which doesn't keep focus or allow tabbing elsewhere.

Overall the accessibility of the data grids is inconsistent at best and inaccessible at worst.

### Impact

Slow to use and inaccessible.

### Additional Context

Honestly, not sure if this should be marked as a feature request or bug. I sort of want to file it as a bug due to the accessibility issues. A note app should be more usable by keyboard.
